code/modules/mob/living/carbon/human/human_attackalien.dm

```diff
@@ -30,7 +30,7 @@
 
 		if(M.a_intent == "disarm")
 			var/randn = rand(1, 100)
-			if (randn <= 80)
+			if (randn <= 100)	// Set to 100 to remove disarm/miss chance

```

Comment:
Just curious, breaks it how? I know its a useless check but if we want to revert or something the variable is right there.